AMP_CACHE_GUIDELINES.md

```diff
@@ -0,0 +1,85 @@
+# AMP Cache Guidelines
+
+## Purpose
+
+Provide guidelines to external parties for implementing their own AMP Caches.
+
+## Guidelines: Core
+
+1. An AMP Cache only serves [valid AMP documents](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md).
```

Comment:
Thanks @jridgewell.  For now this is acceptable. The guidelines are not necessarily mandatory or absolutely prescriptive at this stage so caches may vary slightly in implementation. As the community collaboratively fine-tunes these pointers we should get more clarity on what is considered acceptable.